README.md

```diff
@@ -85,24 +86,41 @@ Three new types are consider safe types under the new contract:
 
     An old SQL
 
-    ```
+    ```ruby
     Obj.where("select * from table where user = #{id}")
     ```
 
-    would need to be rewitten to
+    would need to be rewritten to
 
-    ```
+    ```ruby
     Obj.where(SafeActiveRecord::TrustedSymbol.new(:'select * from table where user
     = ?'), id)
     ```
 
 ## Limitation
 
-`intercept_load` mode is unfortunately not thread safe and could lead to unsafe
+`intercept_load` should be disabled in production system for the following reasons:
+
+* This mode is unfortunately not thread safe and could lead to unsafe
 symbols being treated as trusted in the worst case. It's only a compatible mode
 to support local development that truns off eager load to expedite development
 velocity, and it should never be enabled for production where concurrency is
 controllable by a malicious user.
+* When this mode is enabled, safeActiveRecord is recalculating the symbol table often instead of at initial load, which significantly impacts performance.
```

Comment:
Nitpick: Use "Safe Active Record" or "SafeActiveRecord" as the project name.
```suggestion
* When this mode is enabled, SafeActiveRecord is recalculating the symbol table often instead of at initial load, which significantly impacts performance.
```